clang/test/Sema/aarch64-sve-intrinsics/acle_sve_compact.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu \
+// RUN:   -target-feature +sve -target-feature +sme -target-feature +ssve -target-feature +sme2p2 \
```

Comment:
Does the target feature "+ssve" exist?

That said, given my next comment I think you might be able to drop all the target features other than "+sme".